Referenced by issue #97731, Regression: Python Traceback no longer includes line-number in Info messages
